scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -1070,15 +1099,11 @@ BEGIN
       )
       SELECT ', ST_Transform(' 
             || t.missing_srid_start || t.attname || t.missing_srid_end
-            || ',4326)::Geometry(' 
-            || t.geomtype 
-            || ',4326) AS '
+            || ',4326)::Geometry(GEOMETRY,4326) AS '
             || const.geomcol
```

Comment:
I take it the "enhancement" of retaining the original type restriction turned into a backwards compatibility problem :( There Shall Be No Changes Ever! :)